enterprise/frontend/src/embedding-sdk/components/private/SdkPortalContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from "@emotion/styled";
+
+import { EMBEDDING_SDK_PORTAL_ROOT_ELEMENT_ID } from "embedding-sdk/config";
+
+import { PublicComponentStylesWrapper } from "./PublicComponentStylesWrapper";
+
+/**
+ * This is the portal container used by popovers modals etc, it is wrapped with PublicComponentStylesWrapper
+ * so that it has our styles applied.
+ * Mantine components needs to have the defaultProps set to use `EMBEDDING_SDK_PORTAL_CONTAINER_ELEMENT_ID` as target for the portal
+ */
+export const PortalContainer = () => (
+  <PublicComponentStylesWrapper>
+    <FixedPosition id={EMBEDDING_SDK_PORTAL_ROOT_ELEMENT_ID}></FixedPosition>
+  </PublicComponentStylesWrapper>
+);
+
+const FixedPosition = styled.div`
```

Comment:
could you try removing this special style? I believe it should work. What I expected was only a move of `PortalContainer` inside `enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx`, and nothing else since this style is only needed when the Portal is render before user content.

Reply:
@WiNloSt have you read my [comment here](https://github.com/metabase/metabase/pull/48216#issuecomment-2385409466)?
We can't know where the host app is placing the provider

Reply:
I missed that. This makes sense.